docs/conf.py

```diff
@@ -25,7 +25,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.intersphinx']
+extensions = ['sphinx.ext.intersphinx', 'sphinxcontrib.packetdiag']
```

Comment:
What does this change do?

Reply:
Allows to use packet diagrams. http://blockdiag.com/en/nwdiag/packetdiag-examples.html

Reply:
Oh, that looks neat. :)